src/components/markdown/LanguageSection.tsx

```diff
@@ -5,16 +5,59 @@ import { LANGUAGE_LABELS } from '../../context/UserDataContext/properties/userLa
 
 export const LanguageSection = props => {
   const { lang: userLang } = useContext(UserDataContext);
+const expand = props.expandable === undefined;
 
```

Comment:
props.expandable should be a boolean. So this should probably just be `props.expandable ?? false` or something